MAINTENANCE.md

```diff
@@ -1,34 +1,29 @@
 # Maintainers Guide
 
-This page contains instructions for project maintainers about how our setup
-works, making releases, creating packages, etc.
+> If you want to get involved in the project, you may want to read the
+> [Contributing Guide](CONTRIBUTING.md) instead.
 
-If you want to make a contribution to the project, see the
-[Contributing Guide](CONTRIBUTING.md) instead.
+**This guide documents our procedures and policies for project maintenance
+tasks**, including managing our GitHub repository branches, pull requests,
+continuous integration, releasing, and instructing users on citation practices.
```

Comment:
@MGomezN what do you think? Is this better?

Reply:
It is great! :)